src/org/freedesktop/gstreamer/lowlevel/GType.java

```diff
@@ -98,9 +95,9 @@ public static GType valueOf(long value) {
     public static GType valueOf(String typeName) {
     	GType result = gTypeByNames.get(typeName);
     	if (result == null) {
-    		result = gst.g_type_from_name(typeName);
+    		result = GObjectAPI.GOBJECT_API.g_type_from_name(typeName);
     		if (result.equals(INVALID)) {
-    			// no type has been registered yet
+    			throw new IllegalStateException("No GType registered for this name");
```

Comment:
This changes the behaviour of the function. Is this intended? Before this an unregistered type yielded null, now its an exception.